packages/create-nx-plugin/bin/create-nx-plugin.ts

```diff
@@ -109,8 +109,8 @@ function updateWorkspace(workspaceName: string) {
 
   writeFileSync(nxJsonPath, JSON.stringify(nxJson, undefined, 2));
 
-  removeSync(path.join(workspaceName, 'apps'));
-  removeSync(path.join(workspaceName, 'libs'));
+  rmdirSync(path.join(workspaceName, 'apps'), { recursive: true });
+  rmdirSync(path.join(workspaceName, 'libs'), { recursive: true });
```

Comment:
tbh, I had some issues with `rmdirSync` in the past and especially in Windows. `removeSync` uses `rimraf` behind the scenes and handles more case, so I'm not very keen on replacing 🤔 Actually, I would suggest to replace `rmdirSync` with `removeSync` in all places 🙂

Reply:
i thought it would be ok because `rmDirSync` was already used in @nrwl/tao but i can change them all to removeSync if you want me too

Reply:
@Phillip9587 I would prefer that 🙏 thx

Reply:
@bekos I came up with another solution for the create-nx-plugin package. I first `unlinkSync` the created `.gitignore` files and then `rmdirSync` the `apps` and `libs` dir.

Reply:
@Phillip9587 I see, but tbh I don't feel safe going back to `rmdirSync` from `removeSync` 😞 Looking at `rimraf`'s code there are a ton of cases and especially on Windows that it tries to solve. btw `const retries = isWindows ? 100 : 1` 🙄

Reply:
Done